packages/subgraph/package.json

```diff
@@ -68,6 +69,7 @@
     "deploy:mumbai": "yarn auth && graph deploy --product hosted-service --node https://api.thegraph.com/deploy/ tokensoft/sales-mumbai",
     "deploy:mumbai-next": "yarn auth && graph deploy --product hosted-service --node https://api.thegraph.com/deploy/ tokensoft/sales-mumbai-next",
     "deploy:mumbai-satsuma": "graph deploy sales-mumbai --version-label ${VERSION_LABEL} --node https://subgraphs.alchemy.com/api/subgraphs/deploy --deploy-key ${SATSUMA_DEPLOY_KEY} --ipfs https://ipfs.satsuma.xyz",
+    "deploy:polygon-amoy": "yarn auth && graph deploy --product hosted-service --node https://api.thegraph.com/deploy/ tokensoft/sales-polygon-amoy",
```

Comment:
I think we used Alchemy instead right, do you want to add that deployment command instead?